### PR TITLE
start item timing overlay

### PR DESCRIPTION
**Pull request by aeosynth**
not clear on whether to use inline styles or add a class to column.css

the css could be tweaked (match dotabuff exactly?)

index.html changed during `npm run build`

### CODE REVIEW

File: components/Table/columnDefinitions/matchColumns.jsx [hunk 1]
```diff
@@ -107,13 +107,22 @@ const overviewColumns = [
...
-          />);
+            style={{ position: 'relative' }}
+          >
+            <img
+              style={{ height: 25, margin: '0 3px' }}
+              role="presentation"
+              src={`${API_HOST}${item.img}`}
+            />
+            <span className={styles.timing}>
+              {time / 60 | 0}
```
Comment (howardchung) [inline components/Table/columnDefinitions/matchColumns.jsx]: why | 0 ?

Comment (aeosynth) [inline components/Table/columnDefinitions/matchColumns.jsx]: it's `Math.floor`

File: components/Table/columnDefinitions/matchColumns.jsx [hunk 2]
```diff
@@ -107,13 +107,22 @@ const overviewColumns = [
...
+            </span>
```
Comment (mike-robertson) [inline components/Table/columnDefinitions/matchColumns.jsx]: I think you mean to put `time / 60 || 0`

File: index.html [hunk 3]
```diff
@@ -14,6 +14,6 @@
   </head>
   <body>
     <div id="react"></div>
-    <script src="/build/bundle.js"></script>
+    <script src="/build/313f39d7e16424d48a25.bundle.js"></script>
   </body>
```
Comment (mike-robertson) [inline index.html]: don't change this, it will fail to work when we deploy

File: index.html [hunk 4]
```diff
@@ -14,6 +14,6 @@
   </head>
   <body>
     <div id="react"></div>
-    <script src="/build/bundle.js"></script>
+    <script src="/build/313f39d7e16424d48a25.bundle.js"></script>
```
Comment (howardchung) [inline index.html]: revert this, this should only be done by Travis

**Comment by howardchung** [comment]
signoff

**Pull request merged**